CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/) 
 
+## 20240221
```

Comment:
shall we add al2 ami version and al2023 ami version to denote that we only built these two in this release. the al1 ami version was unchanged: https://github.com/aws/amazon-ecs-ami/blob/main/release-al1.auto.pkrvars.hcl#L1

Reply:
Ah sorry I missed that. Thanks for catching.